extensions/amp-render/1.0/amp-render.js

```diff
@@ -210,9 +213,68 @@ export class AmpRender extends BaseElement {
       'onLoading': () => {
         this.toggleLoading(true);
       },
+      'onDataReady': () => {
+        if (this.element.getAttribute('layout') !== Layout.CONTAINER) {
+          return;
+        }
+        if (!this.getPlaceholder()) {
+          user().warn(TAG, 'placeholder required with layout="container"');
+          return;
+        }
+
+        const height = computedStyle(
+          this.getAmpDoc().win,
+          this.element
+        ).getPropertyValue('height');
+        setStyles(this.element, {
+          'overflow': 'hidden',
+          'height': height,
+        });
+
```

Comment:
Need to calculate height synchronously, since both the async methods below retrieve the height as 0 to support the case where the container cannot resize. Any ideas?